Backpack/src/main/java/net/skyscanner/backpack/rating/internal/RatingSelectors.kt

```diff
@@ -90,13 +91,22 @@ internal class RatingSelectors(
       subtitles = it.resolveStringOrArray(R.styleable.BpkRating_ratingSubtitle)
     }
 
-    val colors = arrayOf(
+    val backgroundColors = arrayOf(
       ColorStateList.valueOf(colorLow),
       ColorStateList.valueOf(colorMedium),
       ColorStateList.valueOf(colorHigh)
     )
-    color = {
-      colors[it.index]
+    backgroundColor = {
+      backgroundColors[it.index]
+    }
+
+    val contentColors = arrayOf(
+      ContextCompat.getColorStateList(context, R.color.bpkWhite)!!,
+      ContextCompat.getColorStateList(context, R.color.bpkSkyGray)!!,
```

Comment:
I think this should technically be black in dark mode

Reply:
This is the colours for the rating score, which is always displayed agains certain colours (equal for LM/DM). Therefore the text colour also remains the same regardless the mode.

Reply:
The reason I think technically it should be black is that we don't use sky-grey in dark-mode.
https://backpack.github.io/guidelines/colour#palette